README.md

```diff
@@ -93,6 +93,15 @@ Interface | `ng g interface my-new-interface`
 Enum      | `ng g enum my-new-enum`
 Module    | `ng g module my-module`
 
+
+
+angular-cli will add reference to `components`, `directives` and `pipes` automatically in the `app.module.ts`. If you need to add this references to another custom module, follow this steps:
+ 
+ 1. `ng g module newModule` to create a new module
+ 2.  call `ng g component new-module/newComponent`
```

Comment:
Can you change `newComponent` and `newModule` to `dash-case` instead? That's how we show these overall in the documentation.